src/primitives/transaction.h

```diff
@@ -356,6 +357,7 @@ class CTransaction
         READWRITE(*const_cast<std::vector<CTxIn>*>(&vin));
         READWRITE(*const_cast<std::vector<CTxOut>*>(&vout));
         READWRITE(*const_cast<uint32_t*>(&nLockTime));
+        // READWRITE(*const_cast<uint32_t*>(&nBlockExpiry));
```

Comment:
Needs to match ZIP (ie. both in position within serialization, and being conditional on version).